e2e/tests/functional/plugins/plot/overlayPlot.e2e.spec.js

```diff
@@ -315,6 +315,31 @@ test.describe('Overlay Plot', () => {
       expect(plotPixelSize).toBeGreaterThan(0);
     }
   );
+
+  test('Can remove an item via the elements pool action menu', async ({ page }) => {
+    const overlayPlot = await createDomainObjectWithDefaults(page, {
+      type: 'Overlay Plot'
+    });
+
+    const swgA = await createDomainObjectWithDefaults(page, {
+      type: 'Sine Wave Generator',
+      parent: overlayPlot.uuid
+    });
+
+    await page.goto(overlayPlot.url);
+    // Wait for plot series data to load and be drawn
+    await waitForPlotsToRender(page);
+    await page.getByTitle('Edit Object').click();
```

Comment:
can we work around the Title?

Reply:
whoop, yeah let me see if I can update this